process/elasticproc/converters/balanceToFloat_test.go

```diff
@@ -69,8 +70,8 @@ func TestComputeBalanceToFloatSliceOfValues(t *testing.T) {
 	ap, _ := NewBalanceConverter(18)
 	require.NotNil(t, ap)
 
-	values := []string{"1000000000000000000", "200000000000000000", "100", "2000"}
-	require.Equal(t, []float64{1, 0.2, 1e-16, 2e-15}, ap.ComputeSliceOfStringsAsFloat(values))
+	values := []string{"1000000000000000000", "200000000000000000", "100", "2000", "0"}
+	require.Equal(t, []float64{1, 0.2, 1e-16, 2e-15, float64(0)}, ap.ComputeSliceOfStringsAsFloat(values))
```

Comment:
could have been `0` instead of `float64(0)`

Reply:
fixed